src/BenchmarkDotNet/Analysers/MultimodalDistributionAnalyzer.cs

```diff
@@ -32,6 +32,6 @@ protected override IEnumerable<Conclusion> AnalyseReport(BenchmarkReport report,
 
         [NotNull]
         private Conclusion Create([NotNull] string kind, double mValue, [CanBeNull] BenchmarkReport report, CultureInfo cultureInfo)
-            => CreateWarning($"It seems that the distribution {kind} (mValue = {mValue.ToString("0.##", cultureInfo)})", report);
+            => CreateWarning($"It seems that the distribution {kind} (mValue = {mValue.ToString("0.##", cultureInfo)}). The issue could be fixed by using more operations.", report);
```

Comment:
The additional sentence can be confused in some cases. Indeed, it's possible to get a false-positive warning about multimodality when the number of operations/invocations/iterations is not big enough. However, in many cases, the performance distribution is multimodal by nature, we can't do anything with it by adjusting benchmark parameters.

Reply:
Good point. I have reverted the change for this warning.